src/store/audio.ts

```diff
@@ -2060,17 +2063,62 @@ export const audioCommandStore = transformCommandStore(
             );
 
             // 読みの内容が変わっていなければテキストだけ変更
-            const isChangedMora = isAccentPhrasesTextDifferent(
+            const isTextDifferent = isAccentPhrasesTextDifferent(
               query.accentPhrases,
               accentPhrases
             );
+            let newAccentPhrases: AccentPhrase[] = [];
+            if (isTextDifferent) {
```

Comment:
`isTextDifferent`、反転させちゃって（isSameTextにしちゃって）ifの最初に処理してしまってもいいかも（今だと最初の処理の方がかなり行数かかっているので）。
なんかこんな感じをイメージしてます。
```suggestion
            const isSameText = isAccentPhrasesTextSame(
              query.accentPhrases,
              accentPhrases
            );
            let newAccentPhrases: AccentPhrase[] = [];
            if (isSameText) {
```